t/plugin/datadog.t

```diff
@@ -395,3 +395,141 @@ message received: apisix\.apisix\.latency:[\d.]+\|h\|#source:apisix,new_tag:must
 message received: apisix\.ingress\.size:[\d]+\|ms\|#source:apisix,new_tag:must,route_name:1,balancer_ip:[\d.]+,response_status:200,scheme:http
 message received: apisix\.egress\.size:[\d]+\|ms\|#source:apisix,new_tag:must,route_name:1,balancer_ip:[\d.]+,response_status:200,scheme:http
 /
+
+
+
+=== TEST 8: testing behaviour with service id
```

Comment:
Is there a test for prefer_name + the route name?

Reply:
Yes. Both enabled: test 2- 6
Prefer_name(true), route_name(false) - test 7
prefer_name(false), route_name(true) - Test-9 (route is created at test-8 with route_name)